lib/find/find.js

```diff
@@ -182,7 +182,7 @@ function find(ctx, next) {
   }
 }
 
-function urlToPath(value, config) {
+function urlToPath(value, config, nodeType) {
```

Comment:
This might as well be called `type` IMO, as it’s a pretty clear jargon in unified-land!

Reply:
Ah, I missed that in your PR body.

> because it is a "Link" not "image", so this looks like a mistake (and prevented tests from working with my code).

It sure looks that way. But, as an image is expected by browsers, they will send something along the lines of an expected `image/png, image/jpg, ...` mimetype, for which the server *could* respond with an image.
I think it’s good to keep it in!